conformance-test-framework/src/main/java/org/jspecify/conformance/ExpectedFact.java

```diff
@@ -0,0 +1,167 @@
+// Copyright 2023 The JSpecify Authors
```

Comment:
Should a new file use `2024` or is it more important that some of the logic in this file existed in 2023 already?

Reply:
This is mostly just extracting code from one file into another. so I kept the copyright year. I see guidance that says not to change the copyright year when moving or copying a file. You could consider this copying-then-editing.

But if someone has more specific guidance, I'm happy to follow it.